lib/convection/model/template/resource_property/aws_elasticsearch_domain_advanced_options.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../resource_property'
+
+module Convection
+  module Model
+    class Template
+      class ResourceProperty
+        # Represents an {https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticsearch-domain.html#cfn-elasticsearch-domain-advancedoptions
+        # Advanced Options Property Type}
+        class ElasticsearchDomainAdvancedOptions < ResourceProperty
```

Comment:
May as well add the third property, indices.fielddata.cache.size, while you're at it even if you don't need it yet: https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-createupdatedomains.html#es-createdomain-configure-advanced-options